examples/eth_provider/test/eth_provider_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule EthProviderTest do
+  use ExUnit.Case
+  doctest EthProvider
+
+  test "the truth" do
+    assert 1 + 1 == 2
+  end
```

Comment:
What is this? 🤣

Reply:
No need for more tests, but if you don't want to have tests, then you can remove the tests, right? 😅
You can have a meaningless test like that, but it adds no value.

Reply:
Done 🫣